styles.css

```diff
@@ -1,3 +1,3 @@
 body {
-  background-color: aquamarine;
+  background-color: #a0a;
```

Comment:
@brytebee this is not the right brand color, kindly check the design to adjust to the right color.

Reply:
@olaishola05 I have made the suggested changes.